### ISSUE TITLE
Inaccurate Stripe decls/ readme

### ISSUE THREAD

**Issue by davidtonghelix** [stripe/react-stripe-elements#139]
### Summary
According to the readme in section https://github.com/stripe/react-stripe-elements#setting-up-your-payment-form-injectstripe, `createToken` takes an object. However, in the flow declaration (src/decls/Stripe.js), it takes 2 params: ```createToken: (
    type: string | ElementShape,
    options: mixed
  )```

They should be made consistent.

**Comment by jez** [comment]
Hey @davidtonghelix!

I think these are referring to two different methods.

The `createToken` that takes just an object is the one exposed to your checkout component by `injectStripe` as `this.props.stripe.createToken`. This is a react-stripe-elements specific function.

The latter `createToken` that takes two parameters is the one that's available from Stripe.js (independent of react-stripe-elements). People using Stripe.js but not react-stripe-elements see something like `elements.createToken`.

TL;DR:

- `this.props.stripe.createToken` takes just an object of options.
- `elements.createToken` takes the element type AND an object of options.

(That's the magic of react-stripe-elements: the by wrapping everything in the `<Elements>...</Elements>` tags, react-stripe-elements automatically populates the parameter number one.)
